docs: Add message formats to contributing.md (#176)
allinbits/cosmos-cash: CONTRIBUTING.md

> Please note we have a code of conduct, please follow it in all your interactions with the project.

Reviewer: Welcome to our community. Please follow our Code of Conduct in all of your interactions with the project.

## Commits messages

> This project follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) format for commit messages and implements it through the [pre-commit](https://pre-commit.com/) framework. Please install this as directed and enable it by running `pre-commit install`. 

Reviewer: ```suggestion
This project follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) format for commit messages and implements it through the [pre-commit](https://pre-commit.com/) framework. You must install and enable pre-commit before you can contribute to the project. 
```

## Pull Request Process

> 1. Install [pre-commit](https://pre-commit.com/) framework and enable it by running `pre-commit install`

Reviewer: ```suggestion
1. To install and enable the [pre-commit](https://pre-commit.com/) framework, run the `pre-commit install` command.
```

> 1. Ensure any install or build dependencies are removed before the end of the layer when doing a
>    build.

Reviewer: ```suggestion
1. Ensure that you remove any dependencies before the end of the layer before you do a build.
```

Reviewer: hmmm what are we doing here? what is the actionable step? how do we remove dependencies? what is "do a build"? hi! and help me understand please.

Author: Hi @noandrea, could you have a look at this when you have a moment? I don't quite understand the intent here. Might need some more detail about how to do this.

Reviewer 2: not applicable to our context, I didn't catch it before when I reviewed the template

> 2. Update the README.md with details of changes to the interface, this includes new environment

Reviewer: ```suggestion
2. Update the project README.md with details of interface changes, including new or changed environment 
   variables, exposed ports, useful file locations, and container parameters.
```

Reviewer: hmmm remove the redundant line below

> 4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you

Reviewer: ```suggestion
4. Pull Requests can be merged after they have approvals of two other developers. 
```

>    do not have permission to do that, you may request the second reviewer to merge it for you.

Reviewer: ```suggestion
   If you do not have merge permissions, request the second reviewer to merge the PR for you.
```

> When making comments against a PR please follow recommendation as per [Conventional Comments](https://conventionalcomments.org/).

Reviewer: ```suggestion
When making comments against a PR, be sure to follow the recommendations in [Conventional Comments](https://conventionalcomments.org/).
```
